### PR TITLE
Fetch estimations if user is logged in

### LINKED ISSUE #2773: userNotAuthorized when accessing 2020 without logging in
example:
https://fra-data.fao.org/assessments/fra/2020/TUR/sections/extentOfForest/

### PR DESCRIPTION
**Pull request by sorja**
image

Fix issue where error `userNotAuthorized` is shown when fetching estimations for tables 1a and 1b

### CODE REVIEW

**Review by rfontanarosa — DISMISSED** [review]

File: src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts [hunk 1]
```diff
@@ -96,10 +97,11 @@ export const useGetTableData = (props: Props) => {
       })
       if (odp) {
         dispatch(DataActions.getOriginalDataPointData({ assessmentName, countryIso, cycleName }))
-        dispatch(
-          DataActions.getNodeValuesEstimations({ assessmentName, countryIso, cycleName, tableName, sectionName })
-        )
+        if (user)
```
Comment (minotogna) [inline src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts]: this should happen if user can edit section data

Comment (sorja) [reply src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts]: Great, updated!

Comment (rfontanarosa) [reply src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts]: Authorizer.canEditData

Comment (minotogna) [reply src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts]: > Great, updated!

I don't see it

Comment (sorja) [reply src/client/pages/AssessmentSection/DataTable/hooks/useGetTableData.ts]: Push didn't go through

**Review by minotogna — APPROVED** [review]

**Review by rfontanarosa — APPROVED** [review]

**Pull request merged**